src/System.Diagnostics.Process/src/System.Diagnostics.Process.builds

```diff
@@ -11,6 +11,9 @@
     <Project Include="System.Diagnostics.Process.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Process.csproj">
+      <TargetGroup>net461</TargetGroup>
+    </Project>
     <Project Include="facade\System.Diagnostics.Process.csproj">
```

Comment:
Yes please lets go ahead and do that merge to keep this consistent. I would also condition all the CS files with the IsPartialFacadeAssembly property instead of both net46 and net461 everywhere.